### PR TITLE
Add "page" argument to CTAs for logging

### PR DESCRIPTION
**Pull request by vdavid**
It was missing from analytics. It was defined in the [figma](https://sourcegraph.com/search?q=context:global+InstallBrowserExtensionCTAShown&patternType=literal) that we wanted the page name sent in the logs.

## Test plan

- Set `eventLogDebug` to `true` in the localStorage, and see if the "page" attributes get logged as CTAs are displayed, in case of IDE and BExt CTAs
- Check out storybook to see if all CTAs still render well. If `eventLogDebug` is `true` then probably the log items will be there even in storybook, and with this, even `NativeIntegrationAlert` can be tested.

### CODE REVIEW

File: client/web/src/repo/actions/BrowserExtensionAlert.tsx [hunk 1]
```diff
@@ -7,13 +7,14 @@ import { eventLogger } from '../../tracking/eventLogger'
...
+    page: 'search' | 'file'
     onAlertDismissed: () => void
 }
 
-export const BrowserExtensionAlert: React.FunctionComponent<Props> = ({ className, onAlertDismissed }) => {
+export const BrowserExtensionAlert: React.FunctionComponent<Props> = ({ className, page, onAlertDismissed }) => {
     useEffect(() => {
-        eventLogger.log('InstallBrowserExtensionCTAShown')
-    }, [])
+        eventLogger.log('InstallBrowserExtensionCTAShown', undefined, { page })
```
Comment (philipp-spiess) [inline client/web/src/repo/actions/BrowserExtensionAlert.tsx:16]: I think you want to set the same arguments for both the second and third parameter: https://sourcegraph.slack.com/archives/CN4FC7XT4/p1643906308345379?thread_ts=1643836277.948139&channel=CN4FC7XT4&message_ts=1643906308.345379

File: client/web/src/repo/actions/BrowserExtensionAlert.tsx [hunk 2]
```diff
@@ -32,6 +33,6 @@ export const BrowserExtensionAlert: React.FunctionComponent<Props> = ({ classNam
     )
 }
 
-const onBrowserExtensionClick = (): void => {
-    eventLogger.log('InstallBrowserExtensionCTAClicked')
+const onBrowserExtensionClick = (page: 'search' | 'file'): void => {
+    eventLogger.log('InstallBrowserExtensionCTAClicked', undefined, { page })
```
Comment (philipp-spiess) [inline client/web/src/repo/actions/BrowserExtensionAlert.tsx:37]: Same here

**Review by philipp-spiess — CHANGES_REQUESTED** [review]
Setting public properties is a bit unexpected but from what I saw at other call sites or this comment in Slack you want to send the same object as the second and third argument: https://sourcegraph.slack.com/archives/CN4FC7XT4/p1643906308345379?thread_ts=1643836277.948139&channel=CN4FC7XT4&message_ts=1643906308.345379

**Review by erzhtor — APPROVED** [review]
LGTM! 👍 

In the meanwhile we are clarifying why event->url is not present in Amplitude, see [slack thread](https://sourcegraph.slack.com/archives/C01LZKLRF0C/p1644490188838879).

**Review by philipp-spiess — APPROVED** [review]
Thanks for quickly fixing :)

File: client/web/src/repo/actions/BrowserExtensionAlert.tsx (hunk 1 above)
Comment (vdavid) [reply client/web/src/repo/actions/BrowserExtensionAlert.tsx]: Thanks, added it for now, but also asked for clarification on why it's needed.

**Pull request merged**